src/core/manager.h

```diff
@@ -335,13 +335,16 @@ struct Manager {
         int private_listen_fd;
         sd_event_source *private_listen_event_source;
 
-        /* Contains all the clients that are subscribed to signals via
-        the API bus. Note that private bus connections are always
-        considered subscribes, since they last for very short only,
-        and it is much simpler that way. */
+        /* Contains all the clients that are subscribed to signals via the API bus. Note that private bus
+         * connections are always considered subscribes, since they last for very short only, and it is
+         * much simpler that way. */
         sd_bus_track *subscribed;
         char **subscribed_as_strv;
 
+        /* The bus id of API bus acquired through org.freedesktop.DBus.GetId. It is used to verify the bus
+         * we shall deserialize subscriptions for is still the same instance as before. */
+        char *bus_id, *deserialized_bus_id;
```

Comment:
why not store it as a sd_id128_t? makes clearer what that is, and removes any ambiguity around uppercase/lowercase